src/main/java/com/urswolfer/intellij/plugin/gerrit/push/GerritPushExtensionPanel.java

```diff
@@ -101,56 +115,87 @@ public void registerGerritPushTargetPanel(GerritPushTargetPanel gerritPushTarget
     public void initialized() {
         initialized = true;
 
-        // force a deferred update (changes are monitored only after full construction of dialog)
```

Comment:
Can you please re-add this comment? I think it is relevant to understand why we did it like this.